system/Debug/Exceptions.php

```diff
@@ -122,6 +125,23 @@ public function exceptionHandler(Throwable $exception)
             ]);
         }
 
+        // For upgraded users.
+        if (! method_exists($this->config, 'handler')) {
+            $this->defaultExceptionHandler($exception, $statusCode, $exitCode);
```

Comment:
Why doesn't this use `ExceptionHandler`? Is it not backwards-compatible behavior?

Reply:
Good point.
It should use ExceptionHandler. 
When using ExceptionHandler, if the behavior changes it is a breaking change.